src/main/java/net/fabricmc/tinyremapper/AsmClassRemapper.java

```diff
@@ -100,7 +99,7 @@ public FieldVisitor visitField(int access, String name, String descriptor, Strin
 			PackageAccessChecker.checkDesc(className, descriptor, "field descriptor", (AsmRemapper) remapper);
 		}
 
-		return super.visitField(access, name, descriptor, signature, value);
+		return new AsmFieldRemapper(super.visitField(access, name, descriptor, signature, value), remapper);
 	}
```

Comment:
Any reason to change the remap logic here? (And some other places in the AsmClassRemapper).

Reply:
just passing the remapper instance

Reply:
just passing the remapper instance

Reply:
I don't get it, the remapper is being passed by the previous implementation as well? It just funneled the remapper instance through the super class field